library/photon

```diff
@@ -12,6 +12,13 @@ Tags: 1.0, 1.0-20181017
 GitFetch: refs/heads/1.0-20181017
 GitCommit: 46235eaadf681be9e5277efd8bc694bf0cb19510
 
+Tags: 3.0, 3.0-20181030
```

Comment:
Did you want to move the `latest` tag  from `2.0` down to `3.0`?

Reply:
@yosifkit Thanks for attention. We will move `latest` to `3.0` later. Current 3.0 is in beta state.